### PR TITLE
Add catppuccin-starlight theme

### PR DESCRIPTION
**Pull request by TheOtterlord**
#### Description

Add catppuccin-starlight theme.

### CODE REVIEW

File: docs/src/content/docs/resources/plugins.mdx [hunk 1]
```diff
@@ -110,6 +110,11 @@ A theme is a Starlight plugin that changes the visual appearance of a site with
 		title="starlight-theme-obsidian"
 		description="Starlight theme inspired by the style of Obsidian Publish sites."
 	/>
+	<LinkCard
+		href="https://github.com/TheOtterlord/catppuccin-starlight"
+		title="catppuccin-starlight"
+		description="📝 Soothing pastel theme for Starlight"
```
Comment (delucis) [inline docs/src/content/docs/resources/plugins.mdx:116]: I know this is the repo description but given nothing else uses an emoji here, I'd suggest we leave it out. For fear of starting an emoji arms race between plugins 😄 

```suggestion
		description="Soothing pastel theme for Starlight"
```

**Review by delucis — APPROVED** [review]
Thanks @TheOtterlord! Just left one small comment, but otherwise LGTM! 😻

File: docs/src/content/docs/resources/plugins.mdx (hunk 1 above)
Comment (TheOtterlord) [reply docs/src/content/docs/resources/plugins.mdx:116]: Haha, fair enough. It's the standard tagline template for catppuccin, so I just copy-pasted.

**Comment by trueberryless** [comment]
Nice to see some new themes coming to life 🧬! 👍

**Pull request merged**